### PR TITLE
Add info on the RakuAST project

### PR DESCRIPTION
**Pull request by tbrowder**
A strawman based on @nine's comments on IRC #raku.

# UPDATE

Feel free to add to this PR.

### CODE REVIEW

**Comment by coke** [comment]
``RaluAST`` appears in a few locations.

**Comment by coke** [comment]
"imformation"

**Comment by tbrowder** [comment]
> @tbrowder I didn't understand the command `False`.

I'm not sure what you are referring to.

**Comment by patrickbkr** [comment]
@tbrowder That wasn't me, but a bot of mine, that has possibly seen some text it interpreted as a command to it. Did your original comment contain something like `{ rcb: }`?

**Comment by tbrowder** [comment]
> Did your original comment contain something like `{ rcb: }

No, but it does have a shell command in a Makefile that has a redirect output  '>' to another file. Would something like that trigger it?

**Comment by patrickbkr** [comment]
@tbrowder It shouldn't. [This](https://github.com/Raku/RakudoCIBot/blob/main/lib/GitHubCITestRequester.rakumod#L132) is the logic doing the command detection. Dunno what triggered it. I guess I'll have to have another look.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ the type and version of the operating system (e.g. `Windows 10`)
 
 Contributions to Rakudo are very welcome.
 
+For those ambitious Raku coders see the [RakuAST](./RaluAST.md) project information.
```
Comment (2colours) [inline CONTRIBUTING.md:18]: I think you meant RakuAST.md

File: README.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 This is Rakudo, a Raku Programming Language compiler for the MoarVM, JVM and Javascript virtual machines.
 
+NOTE: For those ambitious Raku coders see the [RakuAST](./RaluAST.md) project information.
```
Comment (2colours) [inline README.md:5]: Similarly, I think this should be RakuAST.md

Comment (tbrowder) [reply README.md:5]: Yes, good catch. Thanks. I’ll fix that.

**Comment by 2colours** [comment]
For what it's worth, I think this should link to the [RakuAST readme](https://github.com/rakudo/rakudo/tree/rakuast/src/Raku/ast/README.md) some way at least.

File: docs/RakuAST.rakudoc [hunk 3]
```diff
@@ -0,0 +1,19 @@
+=begin pod
+
+head1 RakuAST
+
+Raku branch C<RakuAST> is the future version of the C<master>
```
Comment (vrurg) [inline docs/RakuAST.rakudoc:5]: Actually, it's not the future version of `master` but it is the future `main`.

**Comment by coke** [comment]
@tbrowder - the branch is out of date and has had several unresolved comments (and invalid rakudoc)

Let's clean these up or close out the PR.

**Comment by tbrowder** [comment]
Done.

**Pull request closed without merging**